scripts/build-android.sh

```diff
@@ -14,23 +14,23 @@ echo
 
 GRADLE_PROPERTIES="--daemon --parallel -q -b android/build.gradle"
 
+npm install
+
 case $TARGET in
   debug)
-    echo "Not supported yet."
-    ## lein do clean, cljsbuild once android && ./android/gradlew ${GRADLE_PROPERTIES} assembleDebug
-    ## echo "Generated android/app/build/outputs/apk/app-debug.apk"
-    ## TODO Blocked by https://github.com/status-im/status-react/issues/2669
+    lein do clean, cljsbuild once android && ./android/gradlew ${GRADLE_PROPERTIES} assembleDebug
+    echo "Generated android/app/build/outputs/apk/debug/app-debug.apk"
     exit 1
     ;;
-  prod)
```

Comment:
Does this have any impact on lein prod config stuff or is this completely unrelated? Not sure how TARGET etc is used. Guessing unrelated but just to make sure since "prod" is not that common of an abbrevation for release/production.

Reply:
No it's purely internal to that script (not used by other scripts AFAIK)